consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DataSetResource.java

```diff
@@ -109,7 +108,7 @@ public Response createDataSet(
     public Response describeDataSets(@Context HttpServletRequest request , @QueryParam("dacUserId") Integer dacUserId){
         if (StringUtils.isEmpty(request.getParameter("dacUserId"))) {
             return Response.status(Response.Status.NOT_FOUND).build();
-        }else{
+        } else{
```

Comment:
nit ... halfway there on the spacing:
```suggestion
        } else {
```